core/workspace_svg.js

```diff
@@ -1011,6 +1011,14 @@ Blockly.WorkspaceSvg.prototype.isDragging = function() {
 };
 
 /**
+ * Is this workspace draggable and scrollable?
+ * @return {boolean} True if this workspace may be dragged.
+ */
+Blockly.WorkspaceSvg.prototype.isDraggable = function() {
```

Comment:
Yeah, I'm fine with it being public.  It's a perfectly reasonable utility function.